src/components/EditCareerDevForm.js

```diff
@@ -25,20 +27,23 @@ class SkillsCareerDevForm extends Component {
 
   handleBoolChange = key => value => this.setKey(key, value);
   handleStringChange = key => event => this.setKey(key, event.target.value);
-
+  handleSchoolGoals = key => event => this.setKey(key, event.target.value); 
   handleAttendedIepMeetingChange = attended => {
     this.setKey('attendedIepMeeting', attended);
     if(!attended) this.setKey('iepRole', null);
   };
+
   handleIepRoleChange = this.handleStringChange('iepRole')
   handleHasGraduationPlanChange = this.handleBoolChange('hasGraduationPlan');
+  handlePostSchoolGoals = this.handleSchoolGoals('postSchoolGoals'); 
```

Comment:
Should be `handlePostSchoolGoals = this.handleStringChange('postSchoolGoals');`

The handleStringChange returns a closure that will change one of the variables on the EditCareerDevForm with the same name as the `key` string. Admittedly, I've never seen this style before.